src/data/markdown/translated-guides/en/06 Testing Guides/04 Running large tests.md

```diff
@@ -58,15 +57,17 @@ Memory consumption heavily depends on your test scenarios. To estimate the memor
 run the test on your development machine with 100VUs and multiply the consumed memory by the target number of VUs.
 
 Simple tests will use ~1-5MB per VU. (1000VUs = 1-5GB).
-Tests that are using file uploads can consume tens of megabytes per VU.
+Tests that are using file uploads, or load large JS modules, can consume tens of megabytes per VU.
+Keep in mind that each VU has a copy of all JS modules your test uses.
+If you need to share memory between VUs, consider using [SharedArray](/javascript-api/k6-data/sharedarray/), or an external data store, such as [Redis](/javascript-api/k6-experimental/redis/).
 
 ## General advice
 
 ### Make your test code resilient
 
 When running large stress tests, your script can't assume anything about the HTTP response.
 Often performance tests are written with a "happy path" in mind.
```

Comment:
```suggestion
A large test might elicit unexpected behavior from the SUT, so your test script should handle exceptions well.

In testing literature, the correct, predictable response is often called the [_Happy path_](https://en.wikipedia.org/wiki/Happy_path).
A common mistake in performance testing happens when the script assumes that SUT will take the happy path.
```

Reply:
I'd seen this "Happy path" term twice now, and I only just realized that it's an industry term. I think we should define it. Now I'm not the brightest, but even if it's a common term in CS, not all of our readers will come from the same language and educational backgrounds.

Reply:
It's a pretty common term in testing, even outside of load/performance testing.

Instead of defining it here, let's add it to the [glossary](https://k6.io/docs/misc/glossary/)? Then we can link it here, just as we did with SUT.